frontera/contrib/scrapy/schedulers/frontier.py

```diff
@@ -124,11 +134,13 @@ def process_exception(self, request, exception, spider):
     def open(self, spider):
         log.msg('Starting frontier', log.INFO)
         if not self.frontier.manager.auto_start:
-            self.frontier.start()
+            scrapy_kwargs = {'spider': spider}
+            self.frontier.start(**scrapy_kwargs)
```

Comment:
Those callbacks are overrode by the particular spider in order to give custom instructions on how to create a request and how to define which slot to use. A default behaviour is just too much restrictive. These are tasks that in many situations only the spider knows exactly how to handle.